pynest/nest/lib/hl_api_nodes.py

```diff
@@ -199,4 +199,4 @@ def GetLocalNodeCollection(nc):
 
     sps(nc)
     sr("LocalOnly")
-    return spp()
+    return spp()
```

Comment:
This patch should probably be removed from the PR.

Reply:
But this line is present in the master branch, no? I can't find the difference between this file in the nest-dend branch or in the master branch.

Reply:
Blank line at end of file was removed.

Reply:
Please add a newline to the end of the file.

Reply:
Done!